### PR TITLE
Faq init

### PR DESCRIPTION
**Pull request by alejandrogznz**
Created the main /faq GET endpoint which is planned to display a search box, and dynamically filter out questions pulled from the database

Created a /faq/api/questions GET endpoint that pulls all the questions in the database into a single .json file.

Created a /faq/post/ endpoint with plans to be a type of "compose" page for new questions.

### CODE REVIEW

File: app/extensions.py [hunk 1]
```diff
@@ -38,3 +38,12 @@ def print(self):
         print(' - lastname: {}'.format(self.lastname))
         print(' - major: {}'.format(self.major))
         print(' - year: {}\n'.format(self.year))
+
+
+class FBQuestion:
+    def __init__(self, qud: str, question_info: dict):
+        self.qid = qid
```
Comment (josueal1) [inline app/extensions.py:45]: Flake8 detected  issue with var `qud` and `qid`

File: app/extensions.py [hunk 2]
```diff
@@ -29,6 +28,7 @@ def __init__(self, uid: str, user_info: dict):
         self._raw_dict['uid'] = uid
 
     def get_dict(self) -> dict:
+        print (self._raw_dict)
```
Comment (josueal1) [inline app/extensions.py:31]: @alejandrogznz Nit: we should get rid of the print?

**Review by josueal1 — CHANGES_REQUESTED** [review]
Requesting 1 easy fix, 1 suggestion

File: app/extensions.py (hunk 2 above)
Comment (alejandrogznz) [reply app/extensions.py:31]: Yup I can change it right now.

**Review by josueal1 — APPROVED** [review]
lgtm now

**Pull request merged**